benchmarks/Kestrel.Performance/Kestrel.Performance.csproj

```diff
@@ -23,7 +23,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="BenchmarkDotNet" />
+    <PackageReference Include="BenchmarkDotNet" Version="0.10.9" NoWarn="KRB4002" />
```

Comment:
This is a workaround for the lineup not being available until the build passes. I can remove it and Kestrel will just be busted for the next 20 minutes